test/jest/acceptance/iac/iac-output.spec.ts

```diff
@@ -83,6 +83,8 @@ describe('iac test output', () => {
           EOL +
           '✗ Files with issues: 3' +
           EOL +
+          '  Invalid files: 1' +
+          EOL +
           '  Ignored issues: 8' +
           EOL +
           '  Total issues: 28 [ 0 critical, 4 high, 8 medium, 16 low ]',
```

Comment:
line 90: 
can we not assert on number of issues, please? this causes flakiness from time to time when we update policies, etc. 
It's also not the goal of this test I believe.
```suggestion
          '  Total issues:',
```

Reply:
(even though this change is not part of this PR, I didn't know where to comment on)

Reply:
It's technically a part of what is being tested here. Apart from the format of the issues summary line, it tests the fact that the number of issues found for each severity adds up to the number of total issues found.

As a compromise, we could use a regular expression that simply ensures the text has numeric values in the relevant locations. However, a regular expression check for these cases would cost more in terms of resources.

What do you think?

Reply:
I think the best scenario would be to update all tests in all places, but that's a seperate discussion. That's why I suggest these tests to go out without numbers and not fail suddently at some other point in time

Reply:
Looking at the rest of the tests in this suite, most would break in the event of any changes to our policies, as we also test the output of the issues list. I'm happy to make the requested change, but I'm not sure it'd be enough from preventing the output tests from breaking in the event of content changes.

Reply:
Yeap anything with numbers that relies on live data will break. I'm just saying let's not continue a pattern we know it's fragile just because we are used to doing it so far. does it make sense?

Reply:
Will do 💯